CMSgov/mint-app
# Remove unused deprecated search queries

## Changes and Description

- Remove  deprecated queries to OpenSearch from the GQL schema
   - underlying resolver code left in the code base in case
   - This removes alerts in VScode about unreachable code



## How to test this change
1. Verify that the code base is not impacted by removing these queries
2. Verify that no other references need to be removed from the code base

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
